packages/apollo-server-env/src/polyfills/fetch.ts

```diff
@@ -0,0 +1,114 @@
+import fetch, { Request, Response, Headers } from 'node-fetch';
+
+Object.assign(global, { fetch, Request, Response, Headers });
```

Comment:
I've been going back and forth on this, so definitely open to changing it. The reason for making them global was that this is how environments other than Node (browser, Cloudflare, fly.io) expose them.

We also have existing code (including `apollo-engine-reporting`) relying on the types being global (because it compiles with the `dom` lib), so this would require those to also import from `apollo-server-env`.